src/anbox/platform/sdl/platform.h

```diff
@@ -97,6 +98,16 @@ class Platform : public std::enable_shared_from_this<Platform>,
   bool window_size_immutable_ = false;
   std::uint32_t focused_sdl_window_id_ = 0;
   Configuration config_;
+
+  enum{MAX_FINGERS = 10};
+  enum{MAX_TRACKING_ID = 10};
+  int touch_slots[MAX_FINGERS];
+  int last_slot = -1;
+  int find_touch_slot(int id);
```

Comment:
Methods and member variables have to be separated with an empty line in between,